webapp/src/main/java/com/box/l10n/mojito/entity/ThirdPartyTranslations.java

```diff
@@ -0,0 +1,39 @@
+package com.box.l10n.mojito.entity;
+
+import javax.persistence.*;
+
+/**
+ * Entity used to store the mapping between third party translation ids and mojito text units.
+ *
+ * @author chidinmae
+ */
+@Entity
+@Table(name = "third_party_translations")
```

Comment:
so `third_party_translations` -> `third_party_text_unit` (we don't use 's' in the table names)